dml_test.go

```diff
@@ -210,7 +210,7 @@ func Test_doBatchedRequestsForCollection(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if err := doBatchedRequestsForCollection(tt.args.auth, tt.args.method, tt.args.url, tt.args.batchSize, tt.args.recordMap); (err != nil) != tt.wantErr {
+			if _, err := doBatchedRequestsForCollection(tt.args.auth, tt.args.method, tt.args.url, tt.args.batchSize, tt.args.recordMap); (err != nil) != tt.wantErr {
```

Comment:
what about this

```suggestion
			if _, err := doBatchedRequestsForCollection(tt.args.auth, tt.args.method, tt.args.url, tt.args.batchSize, tt.args.recordMap); !error.Is(err, tt.wantErr) {
```

here and everywhere

Reply:
I'm not following on that one. In fact it's not something I modified, but anyway, `wantErr` is a boolean, and I guess you want to output the error if you define it as true in your test above? But I don't get the change

Reply:
Oh, I totally missed that, sorry, indeed.

maybe you could code a helper to reuse it, something simple, as any test framework would do.

but maybe something like this

```go
err := doBatchedRequestsForCollection(tt.args.auth, tt.args.method, tt.args.url, tt.args.batchSize, tt.args.recordMap);
checkError(t, tt.wantErr, err, "doBatchedRequestsForCollection")
```

```go
func checkError(t *testing.T, wantErr bool, err error, method string) {
    t.Helper()
   
    if wantErr && err != nil {
      return
    }

    if !wantErr && err == nil {
      return
    }

   t.Errorf("%s() error = %v, wantErr %v", method, err, tt.wantErr)
}
```

But as there is no errors, there might have no need

Reply:
@ccoVeille I'll have to look into that, thank you. It would have to be part of a different PR though. Feel free to open an Issue or Discussion post about it. 

I basically just used the official VS Code extension to generate the test framework, but I'm always down for improving tests.

Reply:
I created 

- #46

Reply:
@ccoVeille Thanks!